types/vis/index.d.ts

```diff
@@ -160,7 +160,7 @@ export interface TimelineOptions {
   itemsAlwaysDraggable?: boolean;
   locale?: string;
   locales?: any; // TODO
-  moment?(): void; // TODO
+  moment?(date: Date): Date;
```

Comment:
According to the docs, it is a typed as a function:
http://visjs.org/docs/timeline/